src/agent/src/policy.rs

```diff
@@ -107,6 +107,35 @@ pub async fn do_set_policy(req: &protocols::agent::SetPolicyRequest) -> ttrpc::R
         .map_err(|e| ttrpc_error(ttrpc::Code::INVALID_ARGUMENT, e))
 }
 
+/// PolicyCopyFileRequest is very similar to CreateContainerRequest from src/libs/protocols, except:
+/// - It wraps the base CreateContainerRequest.
+/// - It has an env_map field which is a map of environment variable names to values.
+/// This makes it easier to validate the environment variables in the policy.
```

Comment:
can we improve language here, "in the policy" sounds a bit confusing?

also would we rather want to say: "which is a mapping of environment variable names to expanded values"?